src/bug-filing/azure-boards/create-azure-boards-issue-filing-url.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { title } from '../../content/strings/application';
+import { getIssueDetailsHtml } from '../common/get-issue-details-html';
+import { HTTPQueryBuilder } from '../common/http-query-builder';
+import { IssueDetailsGetter } from '../common/issue-details-getter';
+import { IssueFilingUrlStringUtils, IssueUrlCreationUtils } from '../common/issue-filing-url-string-utils';
+import { AzureBoardsBugFilingSettings } from './azure-boards-bug-filing-service';
+
+const buildTags = (createBugData: CreateIssueDetailsTextData, standardTags: string[]): string => {
+    const tags = ['Accessibility', title, `rule:${createBugData.ruleResult.ruleId}`, ...standardTags];
+    return tags.join('; ');
+};
+
+export const createAzureBoardsIssueFilingUrlProvider = (
+    stringUtils: IssueUrlCreationUtils,
+    issueDetailsGetter: IssueDetailsGetter,
+    queryBuilderProvider: () => HTTPQueryBuilder,
+) => {
+    return (settingsData: AzureBoardsBugFilingSettings, bugData: CreateIssueDetailsTextData, environmentInfo: EnvironmentInfo) => {
+        const standardTags = stringUtils.standardizeTags(bugData);
+
+        const titleField = stringUtils.getTitle(bugData);
+        const tags = buildTags(bugData, standardTags);
+        const body = issueDetailsGetter(stringUtils, environmentInfo, bugData);
+
+        let bodyField: string = '[Microsoft.VSTS.TCM.ReproSteps]';
+
```

Comment:
For me, this blank line makes it harder to follow that lines 29-33 are about setting `bodyField`.

Reply:
Not sure what's the suggestion here. I could remove the extra line but in my case, I usually prefer to have an empty line before "block-ed" pieces of code, live `if`'s and `for`'s. I guess is something more of a personal choice.